Tribler/community/tunnel/tunnel_community.py

```diff
@@ -148,7 +148,6 @@ def on_ip_address(ip_address):
                     except Exception as exception:
                         self.tunnel_logger.error("Failed to write data to transport: %s. Destination: %r",
                                                  exception, destination)
-                        raise
```

Comment:
1. This was existing logic, but I agree that this can be changed. to be included in the try catch
2. Let me check what this can throw and refactor it
3. ok.